docs/static/monitoring-apis.asciidoc

```diff
@@ -598,46 +736,26 @@ curl -XGET 'localhost:9600/_node/hot_threads?human=true'
 
 Example of a human-readable response:
 
-[source,js]
+["source","js",subs="attributes"]
 --------------------------------------------------
  ::: {}
- Hot threads at 2017-01-12T12:10:15-08:00, busiestThreads=3: 
+ Hot threads at 2017-06-06T18:31:17-07:00, busiestThreads=3: 
  ================================================================================
- 1.02 % of cpu usage, state: timed_waiting, thread name: 'LogStash::Runner' 
```

Comment:
again...it's nice to have the date updated. :-) It's pretty easy on my end to drop in the new examples.